tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -180,6 +180,11 @@ qos/test_qos_sai.py::TestQosSai::testQosSaiPgSharedWatermark[None-wm_pg_shared_l
     conditions:
       - "platform in ['x86_64-arista_7050cx3_32s']"
 
+qos/test_qos_sai.py::TestQosSai::testQosSaiPgHeadroomWatermark:
+  skip:
+    reason: "Priority Group Headroom Watermark is not supported on cisco asic. PG drop counter stat is covered as a part of testQosSaiPfcXoffLimit"
+    conditions:
+      - "asic_type=='cisco-8000'"
```

Comment:
Use "asic_type in ['cisco-8000']" should be more extensible.

Reply:
Addressed the comment and updated the PR